src/gui/Src/Gui/CPUDisassembly.h

```diff
@@ -169,6 +170,7 @@ public slots:
     MenuBuilder* mMenuBuilder;
     MenuBuilder* mHighlightMenuBuilder;
     bool mHighlightContextMenu = false;
+    const char *clipMsg = "The data has been copied to clipboard!\r\n";

```

Comment:
I guess you're right, so the thing would be to just drop off this feature or to make it consistent...which one would it be?